Packs/PagerDuty/Integrations/PagerDuty/PagerDuty.py

```diff
@@ -177,18 +176,18 @@ def extract_on_call_user_data(users, schedule_id=None):
     )
 
 
-def extract_on_call_now_user_data(users_on_call_now):
+def extract_on_call_now_user_data(users_on_call_now: dict[str, Any]) -> CommandResults:
     """Extract the user data from the oncalls json."""
-    outputs = []  # type: List[Dict]
-    contexts = []  # type: List[Dict]
+    outputs: list[dict] = []
+    contexts: list[dict] = []
     oncalls = users_on_call_now.get('oncalls', {})
 
     for i in range(len(oncalls)):
         output = {}
         context = {}
 
-        data = oncalls[i]
-        user = data.get('user')
+        data: dict = oncalls[i]
```

Comment:
This is true already because of `outputs: list[dict]`, the IDE will pick up on that anyway.